project-words.txt

```diff
@@ -679,4 +679,14 @@ redcircle
 bluecircle
 tatarot
 Tatarot
-Scry
\ No newline at end of file
+Scry
```

Comment:
Can you put these in alphabetical order?

it's causing merge conflicts
@chiefdegen

Reply:
and rebase on main

Reply:
No problem. will do it now.

Reply:
No problem. will do it now.

Reply:
just your new words
I'll retroactively sort the unsorted words later

Reply:
> just your new words I'll retroactively sort the unsorted words later

Done.